source/Calamari/Kubernetes/ResourceStatus/ResourceStatusReportWrapper.cs

```diff
@@ -57,14 +57,7 @@ public bool IsEnabled(ScriptSyntax syntax)
                 return result;
             }
 
-            try
-            {
-                statusReportExecutor.ReportStatus(workingDirectory, commandLineRunner, environmentVars);
-            }
-            catch (Exception e)
-            {
-                return new CommandResult("K8s Resource Status Reporting", 1, e.Message, workingDirectory);
```

Comment:
We need this to set the status code, otheriwse Calamari will report 0 as status code and that is assumed to have succceded in Server

Reply:
Good catch. Thanks! Looks like without a proper exit code the status is "succeed with warnings". I will fix this soon